Makefile

```diff
@@ -128,11 +129,6 @@ install: $(KUSTOMIZE) manifests ## Install CRDs into a cluster
 uninstall: $(KUSTOMIZE) manifests ## Uninstall CRDs from a cluster
 	$< build config/crd | kubectl delete -f -
 
-.PHONY: deploy
-deploy: $(KUSTOMIZE) manifests ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-	cd config/manager && kustomize edit set image controller=${IMG}
```

Comment:
This has not been working since the image name was changed from `controller` to `quay.io/metal3-io/baremetal-operator`. See [this commit](https://github.com/metal3-io/baremetal-operator/commit/209dbd095d8314c5e2fa106c69020fc06c34271e#diff-ce0d597a9e5f20a8fe61c3ba5e6185a6b90cfef2c00bb5b5ef3e97b15b79f480)

Reply:
The kustomize part was not working properly, but I guess everything else was?

Reply:
True. I was assuming that since the kustomize part was broken there was no point in the target as a whole. Should I rather keep the `kubectl apply` part and remove the rest?

Reply:
Can we maybe leave it in place, but make it depend on the new `set-manifest-image-bmo` target instead of running kustomize?

Reply:
Sounds like a good solution. I'll update!

Reply:
Done. Please take a look @dtantsur